src/components/inputs/Button.tsx

```diff
@@ -1,23 +1,31 @@
 import React from 'react';
 import {Button as MuiButton, Icon, Box} from '@mui/material';
+import PropTypes from 'prop-types';
+import {DashComponentProps} from 'props';
 
 /**
  * Button component
  */
-const Button = (props) => {
-  const {
-    id,
-    text,
-    margin,
-    width,
-    nClicks,
-    startIcon,
-    endIcon,
-    setProps,
-    iconColor,
-    ...remainingProps
-  } = props;
-
+const Button = ({
```

Comment:
The props looked cleaner as they were defined before, the Dash TS template  also follows this syntax:
```typescript
/**
 * Component description
 */
const ComponentName = (props: PropsType) => {
    const { id } = props;
...
```

I think we should also keep the initialization of the props at `Button.defaultProps = {};`

Reply:
I think we agreed on keeping the code the way it is, at least for the moment @filiperosa. I'm resolving this conversation